utils/utils.h

```diff
@@ -52,8 +52,10 @@
 
 # define  SUN_PATH_MAX           (sizeof(struct sockaddr_un) - sizeof(unsigned short int)) /*sun_family*/
 
-# define GB_TIME_STRING_BUFLEN  \
-         (4 + 1 + 2 + 1 + 2 + 1 + 2 + 1 + 2 + 1 + 2 + 1 + 6 + 1 +   5)
+# define  GB_METASTORE_RESERVE   10485760   /* 10 MiB reserve for block-meta */
```

Comment:
Can we make this configurable, defaulting to 10 MiB?

Reply:
I do not find this useful ATM. And do we have a case for this where we might want to alter reserved space ?

Does Heketi  provide an option to tune reserved space ?

Reply:
touche! :-)
On the other hand heketi uses a percentage of the raw capacity instead of a constant size :-)